docs/pre-release/reference/vpn.md

```diff
@@ -0,0 +1,153 @@
+# Telepresence and VPNs
+
+## The test-vpn command
+
+You can make use of the `telepresence test-vpn` command to diagnose issues
+with your VPN setup.
+It will guide you through a series of steps to figure out if there are
+conflicts between your VPN configuration and telepresence.
+
+### Prerequisites
+
+Before running `telepresence test-vpn` you should ensure that your VPN is
+in split-tunnel mode.
+This means that only traffic that _must_ pass through the VPN is directed
+through it; otherwise, the test results may be inaccurate.
+
+You may need to configure this on both the client and server sides.
+Client-side, taking the Tunnelblick client as an example, you must ensure that
+the `Route all IPv4 traffic through the VPN` tickbox is not enabled:
+
+<img src="../images/tunnelblick.png" />
+
+Server-side, taking AWS' ClientVPN as an example, you simply have to enable
+split-tunnel mode:
+
+<img src="../images/split-tunnel.png" />
+
+This setting can usually be toggled without having to reprovision the VPN.
+
+### Testing the VPN configuration
+
+To run it, simply:
+
+```console
+$ telepresence test-vpn
+```
+
+The tool will begin by asking you to disconnect from your VPN; ensure you are disconnected then
+press enter:
+
+```
+Telepresence Root Daemon is already stopped
+Telepresence User Daemon is already stopped
+Please disconnect from your VPN now and hit enter once you're disconnected...
+```
+
+Once it's gathered information about your network configuration without an active connection,
+it will ask you to connect to the VPN:
+
+```
+Please connect to your VPN now and hit enter once you're connected...
+```
+
+It will then connect to the cluster:
+
+
+```
+Launching Telepresence Root Daemon
+Launching Telepresence User Daemon
+Connected to context arn:aws:eks:us-east-1:914373874199:cluster/josec-tp-test-vpn-cluster (https://07C63820C58A0426296DAEFC73AED10C.gr7.us-east-1.eks.amazonaws.com)
+Telepresence Root Daemon quitting... done
+Telepresence User Daemon quitting... done
+```
+
+And show you the results of the test:
+
+```
+---------- Test Results:
+❌ pod subnet 10.0.0.0/19 is masking VPN-routed CIDR 10.0.0.0/16. This usually means Telepresence will be able to connect to your cluster, but hosts on your VPN may be inaccessible while telepresence is connected; to resolve:
+        * Move pod subnet 10.0.0.0/19 to a subnet not mapped by the VPN
+                * If this is not possible, ensure that any hosts in CIDR 10.0.0.0/16 are placed in the never-proxy list
+✅ svc subnet 10.19.0.0/16 is clear of VPN
+
+Please see https://www.telepresence.io/docs/latest/reference/vpn for more info on these corrective actions, as well as examples
+
+Still having issues? Please create a new github issue at https://github.com/telepresenceio/telepresence/issues/new?template=Bug_report.md
+ Please make sure to add the following to your issue:
+ * Attach the zipfile resulting from running `telepresence gather_logs`
```

Comment:
I think it would be good to also recommend `telepresence loglevel debug` here, just like we do in the bug report template.